### DISCUSSION TITLE
Current GDevelop version is nowhere to be found

### DISCUSSION THREAD

**Discussion by PascalLadalle** [4ian/GDevelop#3824] [Enhancement ideas (and their technical discussions)]
A forum user reports that the exact version of GDevelop is not included in the exported files, only the 4.0.99 base version is stated.
The exact version should be indicated somewhere.

[source](https://forum.gdevelop.io/t/get-current-gdevelop-version-from-html5-export/38714)

**Comment by 4ian** [comment]
Note that we don't necessarily want this number to be a "GDevelop version" anymore, but more a "file format version". Each time it's bumped, it means that the format has a breaking change.
